clang/test/Sema/attr-model.cpp

```diff
@@ -5,7 +5,7 @@
 // RUN: %clang_cc1 -triple riscv64 -verify=expected,riscv64 -fsyntax-only %s
 // RUN: %clang_cc1 -triple x86_64 -verify=expected,x86_64 -fsyntax-only %s
```

Comment:
It would be great to check nvptx64 as well -- we want to make sure this does not break CUDA compilation which will likely end up using these attributes.

Reply:
I thought nvptx64 was the GPU-side of things, the GPU side is also using code models?

Reply:
nvptx does not *use* the code models, but CUDA compilation will *see* those attribute in the host code in the same TU. The main goal here is to make sure the GPU-side compilation will not be broken.  I guess it may need a separate test under test/SemaCUDA/ where we have things set up to run a CUDA compilation.
You could use RUN lines in ["test/SemaCUDA/function-target.cu"](https://github.com/llvm/llvm-project/blob/d3161defd6b717241a85b6ca528754c747060735/clang/test/SemaCUDA/function-target.cu#L1)  as the base and the same content as you have in clang/test/CodeGen/X86/codemodel.cpp

Reply:
does the change to this test make sense, or am I missing something that putting a copy in test/SemaCUDA would give us?

Reply:
You could do it here, too, as the test does not actually need extra CUDA SDK bits that we have in SemaCUDA/Inputs.

A RUN with  `-x cuda -triple nvptx64-unknown-cuda -fcuda-is-device` should be sufficient.

Reply:
done